proto/storage/cve.proto

```diff
@@ -331,3 +331,9 @@ message CVSSV3 {
   float score = 12;
   Severity severity = 13;
 }
+
+message ImageCVEInfo {
+  string id = 1; // @gotags: search:"CVE" sql:"pk"
```

Comment:
You are probably going to want to use a different search tag.  I fear the framework will be greedy and not necessarily hit the correct table since this tag is also used on the cve table itself.